### PR TITLE
add bagde

### PR DESCRIPTION
**Pull request by harshkotkar5557**

### CODE REVIEW

File: bagde/bagde.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.section-main-text {
+    margin-bottom: 3rem;
+}
+
+.badge-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
```
Comment (vaibhav18matere) [inline bagde/bagde.css:16]: can make separate class for it

File: bagde/bagde.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    position: absolute;
+    height: 1.4rem;
+    min-width: 1rem;
+    top: -10px;
+    left: 54%;
+    font-weight: var(--font-bold);
+    padding: 0.5rem;
+    border-radius: var(--border-full);
+    background: tomato;
+    color: var(--white);
```
Comment (vaibhav18matere) [inline bagde/bagde.css:46]: can use css variables for it

**Review by vaibhav18matere — APPROVED** [review]

File: bagde/bagde.css (hunk 1 above)
Comment (harshkotkar5557) [reply bagde/bagde.css:16]: okay i will do it

File: bagde/bagde.css (hunk 2 above)
Comment (harshkotkar5557) [reply bagde/bagde.css:46]: okay

**Review by Mahi24-prog — APPROVED** [review]
Good to merge

File: bagde/bagde.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            </ul>
+        </header>
+    </nav>
+    <section class="middleSection">
+        <div class="siderbar">
+            <ul class=" d-flex align-center">
+                <li class="sliderbarItem flex-1"><a href="../gettintstarted.html">Installation</a></li>
+                <li class="sliderbarItem flex-1 "><a href="../alert/alert.html">Alerts</a></li>
+                <li class="sliderbarItem flex-1"><a href="../avatar/avatar.html">Avatar</a></li>
+                <li class="sliderbarItem flex-1 active_item"><a href="./bagde.html">Bagdes</a></li>
```
Comment (deekshithmd) [inline bagde/bagde.html:35]: badges

**Review by aayushjain-code — COMMENTED** [review]
Loogs good from my side 🚀

**Pull request merged**